src/job/jobManager.ts

```diff
@@ -116,8 +116,9 @@ export class JobManager {
     try {
       this._jobHandler = null;
       if (job?.payload) {
+        const excludeRepoFromBenchmarks = ['mms-docs'].includes(job.payload.repoName);
```

Comment:
Do we know if this is limited to `mms-docs`, or should this be for any old-gen properties?

Reply:
I'm not certain on if `ruby` or `php` driver docs will fail in the same way - I think those are the only old-gen exceptions, now that server manual isn't actively deploying docs built with old-gen?

Reply:
Gotcha. Down to keep it as-is and alter the array if needed